adm_program/modules/dates/dates_new.php

```diff
@@ -376,6 +378,15 @@ function setLocationCountry() {
     'date_deadline', $gL10n->get('DAT_DEADLINE'), $date->getValue('dat_deadline', $gSettingsManager->getString('system_date').' '.$gSettingsManager->getString('system_time')),
     array('type' => 'datetime', 'helpTextIdLabel' => 'DAT_DEADLINE_DESC')
 );
+
+$form->addSelectBoxFromSql('dat_photo_id',
+    'zugeordnetes Photoalbum',
+    $gDb,
+    'select pho_id, pho_name from ' . TBL_PHOTOS,
```

Comment:
You should only show the photo albums of the current organization.

Reply:
erm - cant find how to do this. ?! please help or hint